themes/bootstrap3/templates/RecordDriver/DefaultRecord/explain-line.phtml

```diff
@@ -0,0 +1,36 @@
+<tr>
+  <td class="percentage">
+    <?=$this->localizedNumber($explainElement['percent'], $decimalPlaces)?>%
+  </td>
+  <td>
+    <?php
+    if (count($fieldName) > 1) {
+      echo $this->transEsc('Synonym') . ': [';
+    }
+
+    for ($i = 0; $i < count($fieldName); $i++) {
+      $name = $fieldName[$i];
+      $value = $fieldValue[$i] ?? '';
+      $exactMatch = $explainElement['exactMatch'][$i] ?? '';
+      if (!empty($name)) {
+        $description = $this->transEsc('IndexFieldDescription::' . $name);
```

Comment:
I think this should be `translate` instead of `transEsc`. Otherwise you could end up comparing an escaped value with `$name` that's not escaped (at least in theory), but more importantly using `transEscAttr` below would double-escape the description.

Reply:
fixed